lib/components/molecules/QuoteMedia/QuoteMedia.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+/**
+ *
+ * QuoteMedia
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import type { Node } from 'react';
+import styles from './QuoteMedia.style';
+import type { Props } from './types';
+
+import Heading from '../../atoms/Heading/Heading';
+
+const QuoteMedia = ({
+  className,
+  quoteClassName,
+  quoteAuthorProp,
+  title,
+  quoteContent,
+  quoteProp,
+  HeadingType,
+  children,
+}: Props): Node => (
+  <div className={className}>
+    <Heading HeadingType={HeadingType} className="title">
+      {title}
+    </Heading>
+    <div className={children && quoteProp ? 'col-1-2' : 'col-1'}>
+      {quoteProp && quoteContent(quoteClassName, quoteAuthorProp, quoteProp)}
```

Comment:
You are using Render props for quoteContent, so passing props should not be your responsibility unless you want to pass additional data.